### PR TITLE
updated readme and added ppt

### LINKED ISSUE #7: Add/Link Team to README
It seems a nice idea to add team members to the README file or have a separate file which links to README somehow.

### PR DESCRIPTION
**Pull request by gat786**
I have updated the main ppt to have team details as well as created seperate ppts and linked them to a table in the workshop 1 folder inside the repo

closes: #7

### CODE REVIEW

**Review by born-2learn — APPROVED** [review]
The changes look fine to me.
I guess the others can recheck their social media handles.

**Comment by r1walz** [comment]
Hey @gat786! Thanks for working on this. I'll take care from here on!

**Comment by gat786** [comment]
Lemme know if I can help anyhow

File: README.md [hunk 1]
```diff
@@ -11,3 +11,15 @@ This repository contains all the materials to be distributed through our worksho
 4. Add README.md containing all the resources and how to use that folder
 5. Submit a Pull Request!
 ```
+## Our Team
```
Comment (r1walz) [inline README.md]: We need one more space above here.

File: workshop 01 - Introduction to Quantum Computing/README.md [hunk 2]
```diff
@@ -5,3 +5,14 @@
 A shallow dive into the world of Quantum Computing! We explore what Quantum can
 achieve, what makes it better than it's classical counterpart? After convincing
 you all, we have made a special hands-on experience session, tailored for you :)
+
+This workshop had talks given by our Contributors and the powerpoint presentations for the same are 
+stored as such 
+
+| Topic | Speakers |
```
Comment (r1walz) [inline workshop 01 - Introduction to Quantum Computing/README.md]: Did you plan on separating the numbers? If not, remove the leading `|`.

File: README.md [hunk 3]
```diff
@@ -11,3 +11,15 @@ This repository contains all the materials to be distributed through our worksho
...
+2.| Rohit Ashiwal |[Email](mailto:rohit.ashiwal265@gmail.com) |[Linkedin](https://www.linkedin.com/in/rohit-ashiwal/)| [Github](https://github.com/r1walz)
+3.| Vedant Khairnar | [Twitter](https://twitter.com/VedantKhairnar3)| [Linkedin](https://linkedin.com/in/vedantkhairnar)| [Github](https://github.com/VedantKhairnar) 
+4.| Ricardo Prins |[Twitter](https://twitter.com/thericardoprins) |[LinkedIn](https://linkedin.com/ricprins) |[Github](https://github.com/ricardoprins)
+5.| Pallavi Agarwal| [Twitter](https://twitter.com/mon_nom_pallavi)| [LinkedIn](https://www.linkedin.com/in/pallavi-agrawal-learner/)| [Github](https://github.com/pallaviagrawal1729)
+6.| Debanjana Bhattacharya | [Twitter]() |[LinkedIn](https://www.linkedin.com/in/debanjona-bhattacharjya-0858a4165/)|[Github](https://github.com/DEBANJANAB)
+7.|Ganesh Tiwari | [Twitter](https://twitter.com/ganeshtiwari786)|[LinkedIn](https://linkedin.com/in/gat786)|[Github](https://github.com/gat786)|
+8.| Mursal Furqan | [Facebook](https://www.facebook.com/mursal.furqan)| [LinkedIn](https://www.linkedin.com/in/mursalfurqan/)| [Github](https://www.github.com/mursalfk)
```
Comment (r1walz) [inline README.md]: This looks good thanks! :D

File: workshop 01 - Introduction to Quantum Computing/intro-to-qsharp.ipynb [hunk 4]
```diff
@@ -5,6 +5,8 @@
    "metadata": {},
```
Comment (r1walz) [inline workshop 01 - Introduction to Quantum Computing/intro-to-qsharp.ipynb]: This actually belongs to some other commit which you accidentally wrapped in. Please, remove it through rebasing.

File: README.md [hunk 5]
```diff
@@ -11,3 +11,15 @@ This repository contains all the materials to be distributed through our worksho
 4. Add README.md containing all the resources and how to use that folder
```
Comment (r1walz) [inline README.md:11]: [A note on your avatar]

It seems that your `.gitconfig` is set up wrong. Please, check `user.name` and `user.email` matches your real name (Ganesh Tiwari) and GitHub email (ganesht049@gmail.com).

**Review by r1walz — CHANGES_REQUESTED** [review]
Hey @gat786! Although this PR is great, it has some major mistakes. Please, attend to them. If you need any assistance please approach me.

File: README.md (hunk 1 above)
Comment (gat786) [reply README.md]: Adding the space

File: workshop 01 - Introduction to Quantum Computing/README.md (hunk 2 above)
Comment (gat786) [reply workshop 01 - Introduction to Quantum Computing/README.md]: Removing the leading | _pipe_

File: README.md (hunk 5 above)
Comment (gat786) [reply README.md:11]: alrighty i didnt think of that also i keep confusing my mail accounts.

File: README.md (hunk 3 above)
Comment (gat786) [reply README.md]: yeyey

File: README.md (hunk 1 above)
Comment (r1walz) [reply README.md]: oops did I say space? I meant a `new line`. Sorry.

**Review by pallaviagrawal1729 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (pallaviagrawal1729) [reply README.md]: Can we have space between Sr and No. :P

**Comment by r1walz** [comment]
@gat786, any updates on this?

**Review by r1walz — APPROVED** [review]

**Comment by r1walz** [comment]
Nice! Thanks @gat786, it looks good!

**Pull request merged**